.github/workflows/e2e-tests.yml

```diff
@@ -65,7 +65,7 @@ jobs:
       - name: Use Node.js latest
         uses: actions/setup-node@v4
         with:
-          node-version: "*"
+          node-version: "22"
```

Comment:
```   
└─ verdaccio@npm:6.1.2 (via npm:~6.1.2)
      ├─ @verdaccio/auth@npm:8.0.0-next-8.15 (via npm:8.0.0-next-8.15)
      │  └─ @verdaccio/signature@npm:8.0.0-next-8.7 (via npm:8.0.0-next-8.7)
      │     └─ jsonwebtoken@npm:9.0.2 (via npm:9.0.2)
      │        └─ jws@npm:3.2.2 (via npm:^3.2.2)
      │           └─ jwa@npm:1.4.1 (via npm:^1.4.1)
      │              └─ buffer-equal-constant-time@npm:1.0.1 (via npm:1.0.1)
      └─ @verdaccio/signature@npm:8.0.0-next-8.7 (via npm:8.0.0-next-8.7)
```

`buffer-equal-constant-time` imported `SlowBuffer` from the `node:buffer`. The `SlowBuffer` has been removed in Node.js 24.
https://github.com/salesforce/buffer-equal-constant-time/blob/c4f462b812870d3bf16606e6fdcf4e04da005a1e/index.js#L4

Reply:
It has been fixed in `jwa`: https://github.com/auth0/node-jwa/pull/53, I will revert the node 22 pin.